src/figuresClasses.ts

```diff
@@ -2,10 +1,74 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Triangle side length must be greater than zero');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
```

Comment:
Similar to the previous comment, use `this.a`, `this.b`, and `this.c` to access the instance properties.